virttest/qemu_storage.py

```diff
@@ -1554,11 +1554,33 @@ def measure(self, target_fmt, size=None, output="human"):
         """
         cmd_list = [self.image_cmd, "measure", ("--output=%s" % output),
                     ("-O %s" % target_fmt)]
+
+        if target_fmt == "luks":
+            target_image = self.params.get("image_measure_target", "tgt")
+            target_image_secret = self.params.get("image_secret_%s" %
+                                                  target_image, "measure")
+            target_image_params = self.params.object_params(target_image)
+            target_image_params["image_format"] = "luks"
+            target_image_params["image_secret"] = target_image_secret
```

Comment:
```suggestion
            self.params["image_format_%s"  % target_image] = "luks"
            target_image_params = self.params.object_params(target_image)
```
We could set the parameters before `object_params`, don't need to set             `target_image_params["image_secret"] = target_image_secret` and `target_image_params["image_format"] = "luks"` again.